endpoints/schemas/commonSchemas.yaml

```diff
@@ -66,7 +66,7 @@ components:
           description: String	Station or network station identification (Channel) 
           type: string  # TODO any constraints?
         mediaOutletReference:
-          description: Buyer external mediaoutlet reference information
+#          description: Buyer external mediaoutlet reference information
```

Comment:
Sample comment: Please keep the description field. Although this description is in the referenced object, it clarifies the intent of this field here.